spring-cloud-huawei-governance/src/main/java/com/huaweicloud/governance/authentication/securityPolicy/SecurityPolicyAccessController.java

```diff
@@ -54,15 +54,12 @@ public boolean isAllowed(AuthRequestExtractor extractor) throws Exception {
 
   private boolean checkDeny(String serviceName, AuthRequestExtractor extractor) {
     if (securityPolicyProperties.matchDeny(serviceName, extractor.uri(), extractor.method())) {
+      // both permissive and enforcing model need print logs(send alarm info).
+      LOGGER.info("[autoauthz unauthorized request] consumer={}, provider={}, path={}, method={}, timestamp={}",
```

Comment:
Please change "unauthorized request" to "[autoauthz] request denied. consumer={} ....."